### PR TITLE
[stdlib] Remove FileCheck from `test_random.mojo`

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
Related to #2024

### CODE REVIEW

**Comment by Moosems** [comment]
Duplicate of #2298?

**Comment by gabrieldemarmiesse** [comment]
@Moosems I'm not sure I understand. What do you mean?

**Comment by Moosems** [comment]
My apologies, I misread `test_unroll` as `test_random` for some reason 🤦‍♂️

File: stdlib/test/random/test_random.mojo [hunk 1]
```diff
@@ -10,55 +10,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
-# RUN: %mojo %s | FileCheck %s
+# RUN: %mojo -debug-level full %s
```
Comment (JoeLoser) [inline stdlib/test/random/test_random.mojo:13]: Similar comment as others about the `-debug-level full` vs `%mojo-no-debug`

**Review by JoeLoser — APPROVED** [review]
Big improvement in readability and clarity compared to the previous `FileCheck` use — thank you for continuing to push on this!  Just one minor comment about the `%mojo-no-debug` vs `-debug-level full` fixup and then I'm happy to land this.

File: stdlib/test/random/test_random.mojo (hunk 1 above)
Comment (gabrieldemarmiesse) [inline stdlib/test/random/test_random.mojo:13]:
```suggestion
# RUN: %mojo-no-debug %s
```

**Pull request merged**